### PR TITLE
fix: rosalinet

### LINKED ISSUE #1039: rosalinet
MyGithubProfile

### PR DESCRIPTION
**Pull request by rosalinet**
resolve #1039

### CODE REVIEW

**Review by kabir0x17 — APPROVED** [review]
Looks good to me 👍

File: profiles/rosalinet.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "image": "https://user-images.githubusercontent.com/92281462/153997518-4ec45e44-543a-45fc-93e6-72a89ae3a85d.png",
+  "issueId": 1038,
```
Comment (Panquesito7) [inline profiles/rosalinet.json:3]: Wrong issue number.
```suggestion
  "issueId": 1039,
```

**Review by CodingSpecies — COMMENTED** [review]
@rosalinet do you need any help? This pr has been open for a while. Thank you.

**Comment by rosalinet** [comment]
I think I just accidentally did it twice. Sorry!

**Pull request closed without merging**